files/securedrop-client

```diff
@@ -7,5 +7,5 @@ cd /opt/venvs/securedrop-client
 # Now let us try to run alembic first
 ./bin/alembic -c /usr/share/securedrop-client/alembic.ini upgrade head
 
-# Now execute the actual client
-./bin/sd-client
+# Now execute the actual client, only if running in an sd-app
+if [ "$(hostname)" = "sd-app" ]; then ./bin/sd-client; else echo "Not running in sd-app, client not starting."; fi
```

Comment:
Same point as above, re: hostname vs qubesdb-read. Using the latter would log an error in a non-Qubes environment, but would still do the right thing. We could also test for `qubesdb-read`, then proceed with comparing.